docs/tutorials/on-device-training/ios-app.md

```diff
@@ -173,34 +174,34 @@ Save the `Podfile` and run the following command to install the dependencies:
 pod install
 ```
 
-This will create a `MyVoice.xcworkspace` file in the project directory. Open the `xcworkspace` file in Xcode. This will open the project in Xcode with the dependencies installed.
+This will create a `MyVoice.xcworkspace` file in the project directory. Open the `xcworkspace` file in Xcode. This will open the project in Xcode with the CocoaPods dependencies available.
 
-Now, create a new group in the project and name it `artifacts`. Drag and drop the artifacts generated in the previous section into the `artifacts` group. Make sure to select `Create folder references` and `Copy items if needed` options. This will add the artifacts to the project.
+Now, right click on the 'MyVoice' group in the project navigator and click 'New Group' to create a new group in the project called `artifacts`. Drag and drop the artifacts generated in the previous section into the `artifacts` group. Make sure to select `Create folder references` and `Copy items if needed` options. This will add the artifacts to the project.
 
-Next, create a new group in the project and name it `recordings`. This group will contain the audio recordings that will be used for training. You can generate the recordings by running the [`recording_gen.py`](https://github.com/microsoft/onnxruntime-training-examples/blob/master/on_device_training/mobile/ios/recording_gen.py) script at the root of the project. Alternatively, you can also any other recordings of length 10 seconds in .wav format.
+Next, right click on the 'MyVoice' group and click 'New Group' to create a new group in the project called `recordings`. This group will contain the audio recordings that will be used for training. You can generate the recordings by running the [`recording_gen.py`](https://github.com/microsoft/onnxruntime-training-examples/blob/master/on_device_training/mobile/ios/recording_gen.py) script at the root of the project. Alternatively, you can also use any other recordings of length 10 seconds in .wav format except the ones that you are planning to use for inference. Make sure to name the recordings as `other_0.wav`, `other_1.wav`, etc. and add them to the `recordings` group.
```

Comment:
regarding user-provided `other_n.wav` files, do they need to be in a particular WAV format? e.g., 16khz mono ...

if so, may be good to include details about the required format. alternatively, we can just ask them to use what recording_gen.py generates.

Reply:
"except the ones that you are planning to use for inference"

should we also say that the recordings should be of other speakers' voices? also for inference the app will record new audio, so maybe we don't need to worry about reusing audio files from training.